en/micro-integrator/docs/references/mediators/enrich-Mediator.md

```diff
@@ -309,12 +309,15 @@ INFO {LogMediator} - {proxy:TestEnrich} Student name is :  = "Tom"
 
 !!! Info
     This feature is currently supported only for JSON.
+    
+!!! Info
+    We can provide comma-separated multiple JSONPath expression for this `remove` operations as in the following example.
 
-In this example, we will remove the `modules` from every student.
+In this example, we will remove the `modules` from every student and the first student in the array.
```

Comment:
can we explain this?
what do we mean 'every student and the first student in the array'?

Reply:
Before the examples, we have given a payload which is commonly used for all the examples. https://ei.docs.wso2.com/en/7.2.0/micro-integrator/references/mediators/enrich-Mediator/#payload
Here I'm referring to that payload. I'm removing the first student from the array and modules from all the students.